From 0fd0bb29bde950161257d711b745244e0180bdf2 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Fri, 22 Mar 2013 20:51:08 -0700
Subject: [PATCH 09/10] ck: set localversion -ck

Imported from: http://ck.kolivas.org/patches/3.0/3.8/3.8-ck1/patches/ck1-version.patch

Add -ck version in inconspicuous place where it will merge relatively easily
with later kernel versions.

-ck

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 Makefile |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index 61e2c24..8fefc34 100644
--- a/Makefile
+++ b/Makefile
@@ -10,6 +10,10 @@ NAME = Unicycling Gorilla
 # Comments in this file are targeted only to the developer, do not
 # expect to learn how to build the kernel reading this file.
 
+CKVERSION = -ck1
+CKNAME = BFS Powered
+EXTRAVERSION := $(EXTRAVERSION)$(CKVERSION)
+
 # Do not:
 # o  use make's built-in rules and variables
 #    (this increases performance and avoids hard-to-debug behaviour);
-- 
1.7.10.4

